client-app/ui-kit/utilities/images.ts

```diff
@@ -0,0 +1,11 @@
+export function loadImage(fileName: string): string {
+  return new URL(`/client-app/assets/images/${fileName}`, import.meta.url).href;
+}
+
+export function loadIcon(name?: string): string {
+  return new URL(`/client-app/assets/icons/basic/${name}.svg`, import.meta.url).href ?? "";
```

Comment:
dependent from client-app.
add task to make it init phase settings. or move all images to ui kit folder

Reply:
https://virtocommerce.atlassian.net/browse/VCST-2494